lib/matplotlib/text.py

```diff
@@ -557,6 +557,9 @@ def draw(self, renderer):
         self._set_gc_clip(gc)
 
         if self._bbox:
+            self._bbox.update(dict(clip_box=self.clipbox,
```

Comment:
What about making the combined dict a local variable, like

```
bbox_params = dict(self._bbox, clip_box=self.clipbox, ...
```

I'm somehow against syncing some state, that is internal and exactly determined by some other state.